src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+}
+
+body {
+  margin: 0;
+  font-family: "Avenir", sans-serif;

```

Comment:
its better to move fonts to the `html` selector

Reply:
Ok.